ansible/roles/liquid-core/templates/local.py

```diff
@@ -19,3 +19,4 @@
 
 HOOVER_APP_URL = 'http://hoover.{{ liquid_domain }}'
 HYPOTHESIS_APP_URL = 'http://hypothesis.{{ liquid_domain }}'
+DOKUWIKI_APP_URL = 'http://dokuwiki.{{ liquid_domain }}'
```

Comment:
Did you document this config parameter in liquid-core?
Did you include the DokuWiki link in the liquid-core home page?
I think it'd be cleaner if we'd merge this PR and the liquid-core one at the same time.